modules/sdk-lib-mpc/test/unit/util.ts

```diff
@@ -75,4 +81,10 @@ describe('mpc utils', () => {
       }
     });
   });
+
+  it('should get derivation path from a provided seed', function () {
+    const seed = '1';
+    const derivationPath = getDerivationPath(seed);
+    assert.strictEqual(derivationPath, 'm/999999/163767512/266960264');
+  });
```

Comment:
Missing tests with `isMaster=false`. 

Also missing tests around some edge cases. What happens when `seed` is an empty string?